core/src/main/scala/kafka/server/FetchSession.scala

```diff
@@ -690,6 +702,10 @@ class FetchSessionCache(private val maxEntries: Int,
     * 2. B is considered "stale" because it has been inactive for a long time, or
     * 3. A contains more partitions than B, and B is not recently created.
     *
+    * Prior to KAFKA-9401, the session cache was not sharded and we looked at all
```

Comment:
This docs is great. Could you please update this also?

https://github.com/apache/kafka/blob/9b8aac22ec7ce927a2ceb2bfe7afd57419ee946c/core/src/main/scala/kafka/server/KafkaConfig.scala#L182